lib/rbs/test/hook.rb

```diff
@@ -27,8 +27,8 @@ module Hook
         :! => "not",
         :<< => "lshift",
         :>> => "rshift",
-        :~ => "tilda",
-        :=~ => "eqtilda",
+        :~ => "tilde",
+        :=~ => "eqtilde",
```

Comment:
Is it okay to rename them? I could not find "tilda" in other places. So I hope nothing will be broken.